### PR TITLE
Fix backslash in shader source

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

File: sdk/tests/deqp/modules/shared/glsShaderLibrary.js [hunk 1]
```diff
@@ -145,9 +145,7 @@ var gluShaderUtil = framework.opengl.gluShaderUtil;
         if (trimFront)
             str = str.replace(/^\s*\n/, '');
         return str
-            .replace('\\n', '\n')
-            .replace('\\t', '\t')
-            .replace(/\\/g, '');
+            .replace(/\\\\/g, '\\');
```
Comment (qkmiao) [inline sdk/tests/deqp/modules/shared/glsShaderLibrary.js]: See C++ code:
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/opengl/gluShaderLibrary.cpp#537

tow \ is converted to one \: "\\\\" -> "\\", e.g. "\\note" should be converted to "\note" other than
 "
ote".

Comment (zhenyao) [inline sdk/tests/deqp/modules/shared/glsShaderLibrary.js]: What about replacing "\n" to '\n' and "\t" to '\t"? That seems lost with this change?

Comment (qkmiao) [inline sdk/tests/deqp/modules/shared/glsShaderLibrary.js]: It's difficult to do replacement for '\n', '\t', '\\' with repalce() at the same time. Align the code with C++.

**Comment by zhenyao** [comment]
lgtm

**Pull request merged**